.travis.yml

```diff
@@ -3,22 +3,17 @@ language: node_js
 notifications:
   email: false
 node_js:
-  - 4
+  - 6
```

Comment:
Does it fail in Node 4? Not a big deal, I planned to suggest to align Node dependency with hapi anyway (which is 8 now), and I think requiring 6 is reasonable, just wondering